packages/plugin-flex/src/commands/flex/plugins/upgrade-plugin.ts

```diff
@@ -101,6 +105,27 @@ export default class FlexPluginsUpgradePlugin extends FlexPlugin {
     flexPlugin,
   ];
 
+  private static packagesToRemoveFlexUI2 = new Map([
```

Comment:
Don't use `Map` - you aren't actually using anything mappy here - so just use `{'react-router': 'react-router-dom', ...}`